README.md

```diff
@@ -9,26 +9,29 @@ Work zone code and documentation for WZDx, iCone, etc.
 # WZDX Translator
 
 ## Project Description
+This is an open source, proof of concept solution for translating work zone data in the form of COtrip/Salesforce, iCone, and NavJOY messages to the standardized WZDx 3.1 format. This project was developed for CDOT. A unique translator has been developed for each of these message types. These translators read in the source message, parse out specific fields, and generate a WZDx 3.1 message. For more information on these message formats and the data mappings between these messages and the WZDx format, see the [documentation](wzdx/docs). sample_files are located [here](wzdx/sample_files). All these translators are built to run from the command line and from GCP cloud functions, hosted within the CDOT OIM WZDX environment, connected to the RTDH (real time data hub). For more information on cloud hosting, see [GCP_cloud_function](wzdx/GCP_cloud_function). 
 
-This is an open source, proof of concept solution for translating work zone data in the form of COtrip/Salesforce, iCone, and NavJOY messages to the standardized WZDx 3.1 format. This project was developed for CDOT. A unique translator has been developed for each of these message types. These translators read in the source message, parse out specific fields, and generate a WZDx 3.1 message. For more information on these message formats and the data mappings between these messages and the WZDx format, see the [documentation](translator/docs). Sample files are located [here](translator/sample%20files). All these translators are built to run from the command line and from GCP cloud functions, hosted within the CDOT OIM WZDX environment, connected to the RTDH (real time data hub). For more information on cloud hosting, see [GCP_cloud_function](translator/GCP_cloud_function). 
-
-## Prerequisites
+## Installation
+```
+pip install wzdx-translator-jacob6838
```

Comment:
Is this package name until we can get CDOT supported name/account to push with?

Reply:
yeah, I can easily change it. It is recommended to put your pypi user name in the package title if it is not official

Reply:
Can we just add to the README a brief description of this, saying the name is temporary and will be changed in the future with a TODO in there for it?

Reply:
We can add this to the backlog as a task to remind us to get the package name swapped over.

Reply:
what the heck, this was minimized. weird